### DISCUSSION TITLE
Error gopkg.in/yaml.v2: module gopkg.in/yaml.v2@latest found (v2.4.0), but does not contain package gopkg.in/yaml.v2

### DISCUSSION THREAD

**Discussion by alhamsya** [redis/go-redis#1977] [Q&A]
i got error 
`gopkg.in/yaml.v2: module gopkg.in/yaml.v2@latest found (v2.4.0), but does not contain package gopkg.in/yaml.v2`

**Comment by vmihailenco** [comment]
We don't have that module in https://github.com/go-redis/redis/blob/master/go.mod
